scout/server/blueprints/cases/templates/cases/case_bionano.html

```diff
@@ -0,0 +1,257 @@
+          {% if case.madeline_info and case.individuals|length > 1 %}
+            {{ pedigree_panel() }}
+          {% else %}
+            <p>No pedigree picture available.</p>
+          {% endif %}
+        </div>
+        <div class="col-md-8">
+          {{ synopsis_panel() }}
+          {{ comments_panel(institute, case, current_user, comments) }}
+        </div>
+      </div> <!-- end of div row -->
+
+      <div class="row">
+        <div class="col-sm-12">{{activity_panel(events)}}</div>
+      </div>
+
+      {{ modal_synopsis() }}
+    </div> <!-- end of card body -->
+  </div> <!-- end of card div-->
+</div>
+{% endmacro %}
+
+{% macro bionano_individuals_table(case, institute, tissues) %}
+<form method="POST" action="{{ url_for('cases.update_individual', institute_id=institute._id, case_name=case.display_name) }}">
+  <div class="card panel-default">
+    <div class="panel-heading" id="fshd_table_heading">FSHD loci - Individuals</div>
+    <div class="table-responsive">
+      <table class="table table-sm" aria-describedby="fshd_table_heading">
+        <thead>
+          <tr>
+            <th data-bs-toggle='tooltip' data-bs-container='body' style="width: 20%"
+             title="Display name of sample">Sample</th>
+            <th data-bs-toggle='tooltip' style="width: 5%" title="Sample sex">Sex</th>
+            <th data-bs-toggle='tooltip' style="width: 5%" title="Phenotype of sample">Phenotype</th>
+            <th data-bs-toggle='tooltip' data-bs-container='body'
+              style="width: 5%" title="Chromosome of D4Z4 locus">Chr</th>
+              <th data-bs-toggle='tooltip' data-bs-container='body'
+                style="width: 5%" title="Haplotype of D4Z4 locus">Haplotype</th>
+            <th data-bs-toggle='tooltip' data-bs-container='body'
+             title="D4Z4 repeat unit count" style="width: 5%">D4Z4 count</th>
+             <th data-bs-toggle='tooltip' data-bs-container='body'
+              title="Repeat spanning coverage" style="width: 5%">Coverage</th>
+            <th data-bs-toggle='tooltip' data-bs-container='body'
+             title="Ancestry prediction from peddy" style="width: 10%">Ancestry (pred.)</th>
+             <th data-bs-toggle='tooltip' data-bs-container='body'
+              title="Tissue origin for the sample" style="width: 15%">Tissue</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for ind in case.individuals %}
+            {% if "fshd_loci" in ind and ind.fshd_loci is not none %}
+              {% for d4z4 in ind.fshd_loci %}
+                <tr {% if d4z4.chromosome == 4 and d4z4.count < 11 and d4z4.haplotype == "4qA" %}
+                      class="bg-danger"
+                    {% elif d4z4.chromosome == 4 and d4z4.count < 11 %}
+                      class="bg-warning"
+                    {% endif %}>
+                  {% if loop.index == 1 %}
+                    <td> {{ ind.display_name }}</td>
+                    <td style="font-weight: bold;">
```

Comment:
On the light mode, text is not very readable on the red and grey lines. Would be better to have white text instead

<img width="1405" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/b8204278-5d99-4bac-9a74-b317bb627210">

Reply:
Almost there with the style. now only the grey rows are not very redable:

<img width="770" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/4f25e60b-8acf-4620-8691-3a2bec3c564b">

Reply:
🙄 lightmoders..  😉